[2022-06-13 00:25:01,511] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: gcs_2_bq_dag.bq_create_yellow_taxi_tripdata_partitioned_table_task scheduled__2022-06-12T00:00:00+00:00 [queued]>
[2022-06-13 00:25:01,542] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: gcs_2_bq_dag.bq_create_yellow_taxi_tripdata_partitioned_table_task scheduled__2022-06-12T00:00:00+00:00 [queued]>
[2022-06-13 00:25:01,545] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 00:25:01,548] {taskinstance.py:1358} INFO - Starting attempt 2 of 2
[2022-06-13 00:25:01,553] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 00:25:01,600] {taskinstance.py:1378} INFO - Executing <Task(BigQueryInsertJobOperator): bq_create_yellow_taxi_tripdata_partitioned_table_task> on 2022-06-12 00:00:00+00:00
[2022-06-13 00:25:01,752] {standard_task_runner.py:52} INFO - Started process 8717 to run task
[2022-06-13 00:25:01,782] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'gcs_2_bq_dag', 'bq_create_yellow_taxi_tripdata_partitioned_table_task', 'scheduled__2022-06-12T00:00:00+00:00', '--job-id', '644', '--raw', '--subdir', 'DAGS_FOLDER/gcs_to_bq_dag.py', '--cfg-path', '/tmp/tmpg9bd6nw3', '--error-file', '/tmp/tmp903uejmo']
[2022-06-13 00:25:01,788] {standard_task_runner.py:80} INFO - Job 644: Subtask bq_create_yellow_taxi_tripdata_partitioned_table_task
[2022-06-13 00:25:02,027] {task_command.py:370} INFO - Running <TaskInstance: gcs_2_bq_dag.bq_create_yellow_taxi_tripdata_partitioned_table_task scheduled__2022-06-12T00:00:00+00:00 [running]> on host 0825143a1170
[2022-06-13 00:25:02,289] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gcs_2_bq_dag
AIRFLOW_CTX_TASK_ID=bq_create_yellow_taxi_tripdata_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-12T00:00:00+00:00
[2022-06-13 00:25:02,300] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-06-13 00:25:02,307] {base.py:68} INFO - Using connection ID 'google_cloud_default' for task execution.
[2022-06-13 00:25:02,314] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-06-13 00:25:02,317] {bigquery.py:2148} INFO - Executing: {'query': {'query': 'CREATE OR REPLACE TABLE trips_data_all.yellow_taxi_tripdata                 --PARTITION BY DATETIME_TRUNC(tpep_pickup_datetime, MONTH)                     AS                     SELECT *                     REPLACE(NULL AS airport_fee)                FROM trips_data_all.yellow_taxi_tripdata_external_table;', 'useLegacySql': False}}
[2022-06-13 00:25:02,371] {bigquery.py:1538} INFO - Inserting job ***_gcs_2_bq_dag_bq_create_yellow_taxi_tripdata_partitioned_table_task_2022_06_12T00_00_00_00_00_4a003c3c4df7da0688ef872856fb3123
[2022-06-13 00:25:04,523] {taskinstance.py:1890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2149, in execute
    job = self._submit_job(hook, job_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2114, in _submit_job
    retry=self.result_retry,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 439, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1544, in insert_job
    job.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1499, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1489, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 137, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 No column definitions in CREATE TABLE at [1:1]

Location: southamerica-east1
Job ID: airflow_gcs_2_bq_dag_bq_create_yellow_taxi_tripdata_partitioned_table_task_2022_06_12T00_00_00_00_00_4a003c3c4df7da0688ef872856fb3123

[2022-06-13 00:25:04,870] {taskinstance.py:1401} INFO - Marking task as FAILED. dag_id=gcs_2_bq_dag, task_id=bq_create_yellow_taxi_tripdata_partitioned_table_task, execution_date=20220612T000000, start_date=20220613T002501, end_date=20220613T002504
[2022-06-13 00:25:04,992] {standard_task_runner.py:97} ERROR - Failed to execute job 644 for task bq_create_yellow_taxi_tripdata_partitioned_table_task (400 No column definitions in CREATE TABLE at [1:1]

Location: southamerica-east1
Job ID: airflow_gcs_2_bq_dag_bq_create_yellow_taxi_tripdata_partitioned_table_task_2022_06_12T00_00_00_00_00_4a003c3c4df7da0688ef872856fb3123
; 8717)
[2022-06-13 00:25:05,322] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-13 00:25:05,459] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
